docs/config_search.md

```diff
@@ -14,88 +14,160 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
+# Table of Contents
+
+- [Search Modes](#search-modes)
+- [Default Search Mode](#default-search-mode)
+- [Brute Search Mode](#brute-search-mode)
+  - [Automatic Brute Search](#automatic-brute-search)
+  - [Manual Brute Search](#manual-brute-search)
+- [Quick Search Mode](#quick-search-mode)
+- [Ensemble Model Search](#ensemble-model-search)
+- [Multi-Model Search Mode](#multi-model-search-mode)
+
+<br>
+
 # Model Config Search
 
-Model Analyzer's `profile` subcommand supports multiple modes when searching to find the best model configuration.
+## Search Modes
+
+Model Analyzer's `profile` subcommand supports multiple modes when searching to find the best model configuration:
+
+- [Brute Force Search](config_search.md#brute-search-mode)
+  - **Search type:** Brute-force sweep of the cross product of all possible configurations
+  - **Default for:**
+    - Single non-ensemble models
+    - Multiple models being profiled sequentially
+  - **Command:** `--run-config-search-mode brute`
+- [Quick Search](config_search.md#quick-search-mode)
```

Comment:
Is quick our default yet for normal runs? I'm guessing no since the docs you just added say no. Have we considered changing that?

The reason I bring it up is I'm wondering if quick should come before brute in the docs.

Reply:
It is not, as per a previous discussion, we were worried about backwards computability and breaking other teams scripts that rely on brute force MA by default.